microprofile/5.0/config/3.0/22.0.0.1-TCKResults-java17.adoc

```diff
@@ -0,0 +1,469 @@
+:page-layout: certification 
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Config 3.0.1.
+
+== Open Liberty 22.0.0.1 - MicroProfile Config 3.0.1 Certification Summary (Java 17)
+
+* Product Name, Version and download URL (if applicable):
++
+https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/22.0.0.1/openliberty-runtime-22.0.0.1.zip[Open Liberty 22.0.0.1]
+* Specification Name, Version and download URL:
++
+link:https://download.eclipse.org/microprofile/microprofile-config-3.0.1/microprofile-config-3.0.1.html[MicroProfile Config 3.0.1]
```

Comment:
To be clear, I was trying to say that all the spec links are probably wrong ... all the files
```suggestion
link:https://download.eclipse.org/microprofile/microprofile-config-3.0.1/microprofile-config-spec-3.0.1.html[MicroProfile Config 3.0.1]
```